Highway.cpp

```diff
@@ -13,23 +17,49 @@ void Highway::changeSpeed(int newSpeed)
 
 void Highway::addVehicleInternal(Vehicle* v)
 {
-    assert(false);
+    //assert(false);
 
     /*
     depending on the derived type, call the member function that doesn't evade the cops. 
     do not call `setSpeed`.  Pick a different function.
     */
+
+    if (auto* car = dynamic_cast<Car*>(v))
+    {
+        car->closeWindows();
+    }
+    else if (auto* motorcycle = dynamic_cast<Motorcycle*>(v))
+    {
+        motorcycle->lanesplitAndRace(55);
```

Comment:
This function has a default argument.
do not pass a value into this function call.
- [ ] revise